### PR TITLE
Update docs for SP2021 game

### PR DESCRIPTION
**Pull request by Adimote**
- Adds a new 'Robots' section.
  - Adds layout of the Crane robot
  - Adds layout of the Forklift robot

- Added compass sensor documentation (mostly copied from SR)
- Added encoder sensor documentation.
- Added magnet documentation
- Added documentation for Ultrasound sensor
- Added documentation for Touch sensor
- Added documentation for the Radio

- Removed Power Board as it's not used, moved the 'wait start' to game state.
- Removed the Servo board as it's not used.

### CODE REVIEW

File: content/api/encoder.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Linear encoders are attached to joints that move in a straight line (i.e. raising/lowering the forklift).
+
+Linear encoders measure the distance the joint has moved from its start position, in *metres*.
+
+All encoders are stored in a list called `encoders[]`, you can access the encoder with `encoders[i]`, where `i` is the slot the encoder is plugged into. You can find this number in the [documentation for the robot](../robots/).
+
+```python
+# Get the distance from the start position
+distance_from_start = r.encoders[i].displacement # in Metres
```
Comment (RealOrangeOne) [inline content/api/encoder.md:18]:
```suggestion
distance_from_start = r.encoders[1].displacement # in Metres
```

File: content/api/encoder.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Most moving parts of the robot have encoders attached. There are two kinds of encoders, linear encoders and rotary encoders. You can tell which is which by the joint it is attached to; if the joint rotates it is a Rotary encoder, whilst if it slides, it is a linear encoder.
+
+## Linear Encoders
+
+Linear encoders are attached to joints that move in a straight line (i.e. raising/lowering the forklift).
+
+Linear encoders measure the distance the joint has moved from its start position, in *metres*.
+
+All encoders are stored in a list called `encoders[]`, you can access the encoder with `encoders[i]`, where `i` is the slot the encoder is plugged into. You can find this number in the [documentation for the robot](../robots/).
```
Comment (RealOrangeOne) [inline content/api/encoder.md:14]:
```suggestion
All encoders are stored in a list called `encoders`, you can access the encoder with `encoders[i]`, where `i` is the slot the encoder is plugged into. You can find this number in the [documentation for the robot](../robots/).
```

File: content/api/encoder.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+distance_from_start = r.encoders[i].displacement # in Metres
+```
+
+## Rotary Encoders 
+
+Rotary encoders are attached to joints that rotate (i.e. wheel rotations).
+
+Rotary encoders measure the total rotation the joint has moved from its start position, in *radians*.
+
+All encoders are stored in a list called `encoders[]`, you can access the encoder with `encoders[i]`, where `i` is the slot the encoder is plugged into. You can find this number in the [documentation for the robot](../robots/).
```
Comment (RealOrangeOne) [inline content/api/encoder.md:27]:
```suggestion
All encoders are stored in a list called `encoders`, you can access the encoder with `encoders[i]`, where `i` is the slot the encoder is plugged into. You can find this number in the [documentation for the robot](../robots/).
```

File: content/api/encoder.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+```python
+# Get the total rotation from the start position
+rotation_from_start = r.encoders[i].rotation # in Radians
```
Comment (RealOrangeOne) [inline content/api/encoder.md:31]:
```suggestion
rotation_from_start = r.encoders[1].rotation # in Radians
```

File: content/api/game-state.md [hunk 5]
```diff
@@ -36,3 +36,17 @@ r.zone
 
 During a competition match, a USB drive will be used to tell your robot
 which corner it's in. By default during development, this is `0`.
+
+## Start Signal
```
Comment (RealOrangeOne) [inline content/api/game-state.md:40]: In the simulator, this doesn't really exist.

File: content/api/magnet-board.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+---
+title: Magnet Board API
```
Comment (RealOrangeOne) [inline content/api/magnet-board.md:2]: I don't think it really makes sense to call this a "board".

File: content/api/magnet-board.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+weight: 8
+---
+ 
+The magnet board is only available on the [Crane robot](../../robots/crane).
+ 
+The magnet of the crane will stick to any magnetic objects (especially containers).
```
Comment (RealOrangeOne) [inline content/api/magnet-board.md:8]:
```suggestion
The magnet of the crane will only stick to containers.
```

File: content/api/magnet-board.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+{{% notice warning %}}
+Check the spelling of `energised`, it's in british english, `energized` will not work!
+{{% /notice %}}
+
+You can get the current state of the magnet by getting the value of `energized` like so:
+
+```python
+magnet_on = r.magnet.energised
+```
```
Comment (RealOrangeOne) [inline content/api/magnet-board.md:28]: There's also the very useful `nearby` API which is worth documenting https://github.com/sourcebots/competition-simulator/blob/sb2021/modules/sr/robot/magnet.py#L25

File: content/robots/crane.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+- Slot 1: Trolley linear encoder
+- Slot 2: Hoist linear encoder
+
+## Ultrasound sensor
+
+The crane has one [ultrasound sensor](../../api/ultrasound) pointing downwards from the magnet. The ultrasound sensor is attached to Arduino pin 0.
+
+## Radio
+
+The crane robot has a [radio](../../api/radio) receiver attached to the hook, which can be used to measure the position of the container relative to the hook in a 2D plane.
```
Comment (RealOrangeOne) [inline content/robots/crane.md:34]: It's not really on a 2D plane.

```suggestion
The crane robot has a [radio](../../api/radio) receiver attached to the hook, which can be used to detect which container is below it.
```

File: content/robots/_index.md [hunk 10]
```diff
@@ -0,0 +1,11 @@
+---
+title: Robots
+weight: 4
+---
+
+In this year's competition we have 2 Robots, which are each controlled by their own python file.
+
+- [Crane Robot](crane) which is controlled by `crane.py`.
+- [Forklift Robot](forklift) which is controlled by `forklift.py`.
```
Comment (RealOrangeOne) [inline content/robots/_index.md:9]: Thoughts on how we document what exists on the physical robot?

File: content/api/compass.md [hunk 11]
```diff
@@ -0,0 +1,13 @@
+---
+title: Compass
+weight: 10
```
[the remark this answers is not in the document]
Comment (Adimote) [reply content/api/compass.md:3]: Sure, makes sense. merge #172 and I'll delete them :)

File: content/api/game-state.md (hunk 5 above)
Comment (Adimote) [reply content/api/game-state.md]: ok, in which case its gone!

File: content/api/magnet-board.md (hunk 6 above)
Comment (Adimote) [reply content/api/magnet-board.md]: Done :)

File: content/robots/_index.md (hunk 10 above)
Comment (Adimote) [reply content/robots/_index.md:9]: Ah I didn't know students would be programming the physical robot, I guess it makes sense to add a new page to `robots/`, maybe in a separate PR

File: content/api/magnet.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+magnet_on = r.magnet.energised
+```
+ 
+## Checking if a container is nearby
+
+The magnet can tell you if there is an object in range to be picked up. You can check if any object is in range by accessing the `nearby` property.
+```python
+# Get if a container is nearby
+is_nearby = r.magnet.nearby # True or False
+```
```
Comment (RealOrangeOne) [inline content/api/magnet.md:36]: Method, not property

```suggestion
The magnet can tell you if there is an object in range to be picked up. You can check if any object is in range by accessing the `nearby` method:
```python
# Is there a container nearby?
is_nearby = r.magnet.nearby() # True or False
```
```

**Review by RealOrangeOne — DISMISSED** [review]
Small correction comment, else looks great! Let's get it in and iterate from there.

**Pull request merged**